elf/module.go

```diff
@@ -467,8 +472,14 @@ func (b *Module) closeMaps() error {
 //
 // It doesn't detach BPF programs from cgroups or sockets because they're
 // considered resources the user controls.
+// It also doesn't unpin pinned maps. Use CloseExt and set Unpin to do this.
 func (b *Module) Close() error {
-	if err := b.closeMaps(); err != nil {
+	return b.CloseExt(nil)
+}
+
+// CloseExt takes a map "elf section -> CloseOptions"
```

Comment:
Now that I see this, it's kinda confusing, because it takes a map of elf map sections. Wdyt about:

```
CloseExt takes a map "elf map section -> CloseOptions".
```

Also, can you add:

```
If an elf map section is not mentioned in the map, it won't be unpinned.
```